src/components/ebay-video/video.stories.ts

```diff
@@ -89,6 +89,11 @@ export default {
             description:
                 "True/False to keep or remove volume slider. Default is False",
         },
+        shakaConfig: {
+            description:
+                "The Shaka player configuration object. This allows users to control Shaka player.",
```

Comment:
Can you provide a link so people know where to find the options here?
```suggestion
                "The Shaka player [configuration object](https://shaka-player-demo.appspot.com/docs/api/tutorial-config.html). This allows users to control Shaka player.",
```

Reply:
Good idea! Done 👍